src/newrelic.coffee

```diff
@@ -35,22 +41,32 @@
 #   spkane
 #   cmckni3
 #   marcesher
-#
+#   rosskarchner
 
 _ = require 'lodash'
 diff = require 'fast-array-diff'
 gist = require 'quick-gist'
 mdTable = require('markdown-table')
 moment = require 'moment'
+minimatch = require 'minimatch'
+
+# https://coffeescript-cookbook.github.io/chapters/arrays/removing-duplicate-elements-from-arrays
+Array::unique = ->
```

Comment:
since we already are importing lodash, could you instead use `_.uniq`/`_.uniqBy`/`_.uniqWith`?

Reply:
I've made that change